test-poc/framework/src/main/java/org/keycloak/test/framework/webdriver/ChromeWebDriverSupplier.java

```diff
@@ -0,0 +1,42 @@
+package org.keycloak.test.framework.webdriver;
+
+import org.keycloak.test.framework.injection.InstanceWrapper;
+import org.keycloak.test.framework.injection.LifeCycle;
+import org.keycloak.test.framework.injection.Registry;
+import org.keycloak.test.framework.injection.Supplier;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class ChromeWebDriverSupplier implements Supplier<WebDriver, TestWebDriver> {
+
+    @Override
+    public Class<TestWebDriver> getAnnotationClass() {
+        return TestWebDriver.class;
+    }
+
+    @Override
+    public Class<WebDriver> getValueType() {
+        return WebDriver.class;
+    }
+
+    @Override
+    public InstanceWrapper<WebDriver, TestWebDriver> getValue(Registry registry, TestWebDriver annotation) {
+        ChromeDriver driver = new ChromeDriver();
+        return new InstanceWrapper<>(this, annotation, driver);
+    }
+
+    @Override
+    public LifeCycle getLifeCycle() {
+        return LifeCycle.GLOBAL;
+    }
+
+    @Override
+    public boolean compatible(InstanceWrapper<WebDriver, TestWebDriver> a, InstanceWrapper<WebDriver, TestWebDriver> b) {
+        return a.getValue().getClass().equals(b.getValue().getClass());
+    }
+
+    @Override
+    public void close(WebDriver instance) {
+        instance.quit(); // TODO might be close() - "Close the current window, quitting the browser if it's the last window currently open"
```

Comment:
Let's use `quit()` for now, and also change the Firefox driver to do the same. Calling `quit()` will end the session entirely, which means we can no longer send commands to this WebDriver, which is what we want as this method will be called when all the tests have finished running.

```suggestion
        instance.quit();
```